JPS_Ontology/ontology/ontoplanningregulation/OntoPlanningRegulation.csv

```diff
@@ -30,8 +30,8 @@ requiresPartyWall,Object Property,,,PlanningRegulation,PartyWall,,A relationship
 requiresRoadBuffer,Object Property,,,RoadCategory,RoadBuffer,,A relationship defining the required road buffer for an urban entity which is enforced by a planning regulation.,https://www.theworldavatar.com/kg/ontoplanningregulation/,requires road buffer
 requiresSetback,Object Property,,,PlanningRegulation,Setback,,A relationship defining the required setback of an urban entity which is enforced by a planning regulation.,https://www.theworldavatar.com/kg/ontoplanningregulation/,requires setback
 override,Object Property,,,PlanningRegulation,PlanningRegulation,,This relationship denotes the prioritization of a specific planning regulation over others. It aims to accommodate the diverse hierarchy of regulations observed across different cities.,https://www.theworldavatar.com/kg/ontoplanningregulation/,override
-appliesTo,Object Property,,,PlanningRegulation,https://www.theworldavatar.com/kg/ontozoning/Zone UNION https://www.theworldavatar.com/kg/ontoplot/Plot,,This relationship serves as a link to signify the direct relevance and applicability of the planning regulation to that specific urban entity.,https://www.theworldavatar.com/kg/ontoplanningregulation/,applies to
-appliesTo,Object Property,,,LandUseRegulation,https://www.theworldavatar.com/kg/ontozoning/LandUseType,,This relationship serves as a link to signify the direct relevance and applicability of a land use regulation to that specific land use type.,https://www.theworldavatar.com/kg/ontoplanningregulation/,applies to
+appliesTo,Object Property,,,PlanningRegulation,https://www.theworldavatar.com/kg/ontozoning/Zone UNION https://www.theworldavatar.com/kg/ontoplot/Plot,only,"This relationship serves as a link to signify the direct relevance and applicability of the planning regulation to that specific urban entity.  In the case of land use regulations, they are applicable only for land use types.",https://www.theworldavatar.com/kg/ontoplanningregulation/,applies to
+appliesTo,Object Property,,,LandUseRegulation,https://www.theworldavatar.com/kg/ontozoning/LandUseType,only,"This relationship serves as a link to signify the direct relevance and applicability of the planning regulation to that specific urban entity.  In the case of land use regulations, they are applicable only for land use types.",https://www.theworldavatar.com/kg/ontoplanningregulation/,applies to
```

Comment:
This is still incorrect - the domain of `appliesTo` is still the intersection of `PlanningRegulation` and `LandUseRegulation` (i.e. the empty set). What is missing is an additional line defining `appliesTo` (and this should be the first one of the three), with the `UNION` of `PlanningRegulation` and `LandUseRegulation` as domain, and the `UNION` of the ranges of both the current definitions. The current two lines then just define restrictions. You may also move the comments into the new definition.